org.kde.akregator.json

```diff
@@ -0,0 +1,495 @@
+        "--talk-name=org.kde.JobViewServer",
+        "--talk-name=org.kde.klauncher5",
+        "--talk-name=org.kde.knotify",
+        "--talk-name=org.kde.kwalletd5",
+        "--talk-name=org.kde.kcookiejar5",
+        "--system-talk-name=org.freedesktop.NetworkManager"
+    ],
+
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh --prefix=/app --with-libraries=system",
+                "./b2 -j $FLATPAK_BUILDER_N_JOBS install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2",
+                    "sha256": "8681f175d4bdb26c52222665793eef08490d7758529330f98d3b29dd0735bccc"
+                }
+            ]
+        },
+        {
+            "name": "gpgme",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://gnupg.org/ftp/gcrypt/gpgme/gpgme-1.15.1.tar.bz2",
+                    "sha256": "eebc3c1b27f1c8979896ff361ba9bb4778b508b2496c2fc10e3775a40b1de1ad"
+                }
+            ],
+            "cleanup": [
+                "*.la"
+            ]
+        },
+        {
+            "name": "qtkeychain",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+                "-DLIB_INSTALL_DIR=/app/lib",
+                "-DBUILD_TRANSLATIONS=NO"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/frankosterfeld/qtkeychain/archive/v0.12.0.tar.gz",
+                    "sha256": "cc547d58c1402f6724d3ff89e4ca83389d9e2bdcfd9ae3d695fcdffa50a625a8"
+                }
+            ]
+        },
+        {
+            "name": "akonadi",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DBUILD_TOOLS=OFF",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DDATABASE_BACKEND=SQLITE"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/release-service/21.12.1/src/akonadi-21.12.1.tar.xz",
+                    "sha256":"d43833f1e587aacb758ed42b7431eea908dc6c69cf89183afcd9059d511b08c6"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "sqlite3",
+                    "config-opts": [
+                        "--enable-threadsafe",
+                        "--enable-threads-override-locks"
+                    ],
+                    "build-options": {
+                        "cflags": "-DSQLITE_ENABLE_UNLOCK_NOTIFY=1"
+                    },
+                    "sources": [
```

Comment:
```suggestion
                    "cleanup": [
                        "*.a"
                    ],
                    "sources": [
```

Reply:
Added  below with other sqlite cleanup

Reply:
good call. I had fixed that change locally.